express-server/mock-data.js

```diff
@@ -1,107 +0,0 @@
-export const players = [
```

Comment:
@avoerman Correct me if I am wrong, but it looks like this file is unused since the data was moved to the json files.